bitnami/redis-cluster/values-production.yaml

```diff
@@ -265,7 +265,13 @@ service:
   ##
   annotations: {}
   labels: {}
-
+  ## Set the service type.
+  ## Setting this to LoadBalancer may require corresponding service annotations for loadbalancer creation to succeed.
+  ## Currently supported types are ClusterIP (default) and LoadBalancer
+  type: ClusterIP
+  ## If service.type is LoadBalancer, request a specific static IP address if supported by the cloud provider.
+  ## otherwise leave blank
+  # loadBalancerIP:
```

Comment:
Please add a space below to separate the sections.

Reply:
@miguelaeh added line breaks.